.travis.yml

```diff
@@ -1,15 +1,16 @@
 language: julia
 os:
     - linux
+    - osx
 julia:
     - 0.5
     - nightly
 notifications:
     email: false
-script:
-    - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-    - julia -e 'Pkg.clone(pwd()); Pkg.clone("Winston")'
-    - xvfb-run julia -e 'Pkg.build("Tk"); Pkg.test("Tk"; coverage=true)';
-    # - xvfb-run julia -e 'Pkg.clone(pwd()); Pkg.build("Tk"); Pkg.test("Tk"; coverage=true)';
+#script:
+#    - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+#    - julia -e 'Pkg.clone(pwd()); Pkg.clone("Winston")'
+#    - xvfb-run julia -e 'Pkg.build("Tk"); Pkg.test("Tk"; coverage=true)';
```

Comment:
Yeah I was just trying to figure this out. Is `xvfb` not needed on Mac? You can get it from XQuartz, but idk if that's installed by default on Travis.